File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.py
import heapq
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        min_heap=[]
        for i in range(len(nums)):
            if i < k:
                heapq.heappush(min_heap,nums[i])
            else:
                if min_heap[0]<nums[i]:
                    heapq.heappop(min_heap)
                    heapq.heappush(min_heap,nums[i])
        return min_heap[0]

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> maxHeap = new  PriorityQueue<>();
        int result= 0 ; 
        for(int i = 0; i < nums.length; i++){
            maxHeap.add(-1 * nums[i]); 
        }
        for(int i = 0; i<k ; i++){
        result = maxHeap.poll();
        }
        return -1 * result;
    }
}